drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-backend/pom.xml

```diff
@@ -157,9 +157,27 @@
       <artifactId>kie-dmn-core</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.kie</groupId>
+      <artifactId>kie-dmn-core</artifactId>
+      <classifier>tests</classifier>
+      <scope>test</scope>
+    </dependency>
 
   </dependencies>
 
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.kie</groupId>
+        <artifactId>kie-dmn-core</artifactId>
+        <version>${version.org.kie}</version>
```

Comment:
Same comment as above, this should not be needed

Reply:
@danielezonca 
This is needed because that artifact with the "test" classifier is not declared/managed in parent bom; so, since every dependency must be "managed", I've put it there.